src/Nethermind/Nethermind.Evm/BlobGasCalculator.cs

```diff
@@ -29,14 +29,14 @@ public static ulong CalculateBlobGas(Transaction[] transactions)
         return CalculateBlobGas(blobCount);
     }
 
-    public static bool TryCalculateBlobGasPrice(BlockHeader header, Transaction transaction, out UInt256 blobGasPrice)
+    public static bool TryCalculateBlobGasFee(BlockHeader header, Transaction transaction, out UInt256 blobGasFee)
```

Comment:
@flcl42 WDYT about the rename?

Reply:
Considering https://github.com/ethereum/EIPs/pull/8316, could be:
TryCalculateBlobGasPricePerUnit -> TryCalculateGasFeePerBlobGas
TryCalculateBlobGasPrice -> TryCalculateBlobBaseFee

Reply:
Seems clearer to me, although I'll use `TryCalculateFeePerBlobGas` rather than `TryCalculateGasFeePerBlobGas`